usermods/usermod_v2_rotary_encoder_ui_ALT/readme.md

```diff
@@ -21,26 +17,22 @@ Press the encoder to cycle through the options:
 * Main Color (only if display is used)
 * Saturation (only if display is used)
 
-Press and hold the encoder to display Network Info
-    if AP is active, it will display the AP, SSID and Password
+Press and hold the encoder to display Network Info. If AP is active, it will display the AP, SSID and Password
 
 Also shows if the timer is enabled.
 
 [See the pair of usermods in action](https://www.youtube.com/watch?v=ulZnBt9z3TI)
 
 ## Installation
 
-Copy the example `platformio_override.sample.ini` to the root directory of your particular build and rename it to `platformio_override.ini`.
-
-To activate this alternative usermod, add `#define USE_ALT_DISPlAY` (NOTE: CASE SENSITIVE) to the `usermods_list.cpp` file, or add `-D USE_ALT_DISPlAY` to your `platformio_override.ini` file
+Copy the example `platformio�override.sample.ini` to the root directory of your particular build and rename it to `platformio_override.ini`.
```

Comment:
Typo?

Reply:
No, any typo. The filename in the folder has a non standard ascii in the filename. Should I rename it?

Reply:
`platformio-override.sample.ini` is invalid name.